vortex-fastlanes/Cargo.toml

```diff
@@ -16,6 +16,7 @@ workspace = true
 
 [dependencies]
 arrayref = { workspace = true }
+parquet = { workspace = true }
```

Comment:
why does fastlanes depend on parquet? this looks vestigial

Reply:
You're right -- this is vestigial